libvast/test/query_queue.cpp

```diff
@@ -55,36 +55,36 @@ std::vector<uuid> cands(uint32_t start, uint32_t end) {
 }
 
 // We need to be able to generate queries with random query ids.
-query rand_query() {
-  auto result
-    = query::make_count(caf::actor{}, query::count::estimate, expression{});
+query_context rand_query() {
+  auto result = query_context::make_count(
+    caf::actor{}, query_context::count::estimate, expression{});
   result.id = uuid::random();
   return result;
 }
 
 uuid make_insert(query_queue& q, std::vector<uuid>&& candidates) {
   uint32_t cands_size = candidates.size();
-  auto query = rand_query();
-  REQUIRE_SUCCESS(q.insert(query_state{.query = query,
+  auto qc = rand_query();
```

Comment:
The `rand_query` function should probably be named `make_random_query_context` instead.

`qc` should probably be `query_context` instead.